server/room.ts

```diff
@@ -641,6 +647,10 @@ export class Room {
       return;
     }
     this.lock = data.locked ? decoded.uid : '';
+    if (!this.isChatEnabled && !data.locked) {
```

Comment:
does this enable the chat when the lock is removed? I don't think we should tie locking and chat together, seems like they should be fully independent